worker-example/worker-example-container/src/main/config/cfg~caf~worker~ExampleWorkerConfiguration.js

```diff
@@ -16,6 +16,7 @@
 ({
     workerName: "worker-example",
     workerVersion: "${project.version}",
+    trackProgressMessages: getenv("CAF_WORKER_TRACK_PROGRESS_MESSAGES"),
```

Comment:
The above change will mean that we don't need changes to the JavaScript files for all the workers.